app/views/layouts/_footer.html.erb

```diff
@@ -17,7 +20,7 @@
         <%= link_to t('footer.links.about'), page_path('about')%>
       </div>
       <div class="link">
-        <%= link_to t('footer.links.user-guide'), page_path('docs/guide')%>
+        <%= link_to t('footer.links.help-guide'), 'https://about.opencasebook.org'%>
```

Comment:
Have you tried something like `root_path(subdomain: 'about')`? https://stackoverflow.com/questions/42014992/rails-link-to-passing-a-subdomain

Reply:
If that doesn't work - feel free to merge